### ISSUE TITLE
Screen doesn't fit on iPhone 11 Pro

### ISSUE THREAD

**Issue by rutgervw** [Make-some-Noise/knzb-zwemdiploma#22]
**Describe the bug**
All screen seem to be to high so the content doesn't fit properly

**To Reproduce**
1. Go to the app
2. Click on anything
3. Check the bottom
4. On some screens the navigation is too low and cut off.
5. Some screens have a blue bar over the navigation (seems to be the area that is "swipe-able" to switch apps)
6. Some screen are scrollable which results in content not visible (because the is no indication that the page is scrollable)

**Expected behavior**
Content should fit tot the screen, no cut-off menu, no blue bar over menu, not scrollable if content can fit screen

**Screenshots**
Blue bar over navigation:
![IMG_9487](https://user-images.githubusercontent.com/24732905/79458943-daed0100-7ff2-11ea-9707-331562bc6064.PNG)

Navigation too low / cut-off:
![IMG_9483](https://user-images.githubusercontent.com/24732905/79459046-fb1cc000-7ff2-11ea-8a0e-3bd8210a9ba8.PNG)
![IMG_9484](https://user-images.githubusercontent.com/24732905/79459074-053ebe80-7ff3-11ea-8ecd-1804a3581bb2.PNG)

Content scrollable (which can hide info):
![IMG_9480](https://user-images.githubusercontent.com/24732905/79459178-2a333180-7ff3-11ea-92f8-18a97ef95873.PNG)
![IMG_9481](https://user-images.githubusercontent.com/24732905/79459180-2acbc800-7ff3-11ea-882f-daae86dc16ac.PNG)

![IMG_9485](https://user-images.githubusercontent.com/24732905/79459197-30c1a900-7ff3-11ea-85ab-0bb1df4a2ba1.PNG)
![IMG_9486](https://user-images.githubusercontent.com/24732905/79459199-328b6c80-7ff3-11ea-9e9a-25fada01965d.PNG)

**Smartphone (please complete the following information):**
 - Device: iPhone 11 Pro
 - OS: 13.3.1

**Additional context**
Basically all screen have something off
